ros2/Dockerfile

```diff
@@ -85,21 +86,19 @@ ENV TINI_VERSION v0.9.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /bin/tini
 RUN chmod +x /bin/tini
 
-RUN apt-get update \
-  && apt install -y libgazebo9-dev ros-${ROS_DISTRO}-gazebo-ros-pkgs \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
-
 RUN apt-get update \
   && apt install -y \
     libgazebo9-dev \
     ros-${ROS_DISTRO}-gazebo-ros-pkgs \
+    rti-connext-dds-${RTI_VERSION} \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
+# Make the default workspace somewhere other than /root, and do not copy the examples
+RUN mv /rticommon_config.sh /opt/rti.com/rti_connext_dds-${RTI_VERSION}/resource/scripts/rticommon_config.sh
```

Comment:
We can just fold this into the `RUN` command above.

Reply:
Done